sdk/python/toolchain/poetry.go

```diff
@@ -29,19 +30,58 @@ type poetry struct {
 
 var _ Toolchain = &poetry{}
 
+const minPoetryVersion = "1.8.0"
```

Comment:
can't this just be created as a `semver.Version`?

Reply:
Yes, thanks!